sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/exception/ResteasyReactiveViolationExceptionMapperTest.java

```diff
@@ -0,0 +1,22 @@
+package io.stargate.sgv2.docsapi.api.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.quarkus.hibernate.validator.runtime.jaxrs.ResteasyReactiveViolationException;
```

Comment:
Not sure we can do anything about this, but this like an internal package shaded (repackage in different Java package) by Quarkus for its own use, originally from RESTeasy's public API.
So just wondering why that would be exposed by Quarkus. If it does I guess we don't have much choice than refer to it, just seems bit off.

Reply:
Yes wrote it in the comment above, you might be right and this might be wrong.. Eric can map it back to the original exception or the other one which is a parent class of this one.

Reply:
Sounds good.